Kconfig

```diff
@@ -43,6 +43,12 @@ comment "Some configuration options may not be here. Use CFLAGS instead."
 comment "!! ERROR: There are conflicting modules active !!"
     depends on ERROR_MODULES_CONFLICT != ""
 
+config RIOT_CI_BUILD
+    bool "CI build"
+    help
+        Indicates that the CI is building. This may add additional modules
+        for the sake of testing.
```

Comment:
What about taking it from the environment?
```
config RIOT_CI_BUILD
    bool
    default y if '$(RIOT_CI_BUILD)' = '1'
    help
        Indicates that the CI is building. This may add additional modules
        for the sake of testing.
```